apps/emqx/src/emqx_types.erl

```diff
@@ -163,6 +163,7 @@
     receive_maximum => non_neg_integer(),
     expiry_interval => non_neg_integer(),
     atom() => term()
+    % transport_started_at => integer(),
```

Comment:
Forgotten code?

Reply:
commented out for bpapi spec backward compatibility.
I will add the note